elements/pl-matrix-component-input/pl-matrix-component-input.py

```diff
@@ -25,6 +25,12 @@ def prepare(element_html, data):
     optional_attribs = ['weight', 'label', 'comparison', 'rtol', 'atol', 'digits', 'allow-partial-credit', 'allow-feedback', 'allow-fractions', 'rows', 'columns', 'allow-blank', 'blank-value']
     pl.check_attribs(element, required_attribs, optional_attribs)
     name = pl.get_string_attrib(element, 'answers-name')
+
+    if name in data['answers_names']:
+        raise Exception('duplicate answer-name attribute: %s' % name)
```

Comment:
Best to avoid raising the base `Exception` when possible. Also, should use f-strings.

```suggestion
        raise ValueError(f'duplicate answer-name attribute: {name}')
```